frontend/src/components/VideoCarrousel.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import '../style/video.css';
+import {
+    ButtonBack, ButtonFirst, ButtonLast, ButtonNext,
+    CarouselProvider, DotGroup, Image, ImageWithZoom, Slide, Slider,
+} from "pure-react-carousel";
+import 'pure-react-carousel/dist/react-carousel.es.css';
+import s from '../style/style.scss';
+
+const VideoCarrousel = ({videos, handleVideoSelect}) => {
+
+    const slider = videos.map((video,vIndex) => {
+    return <Slide index={vIndex}>
+        <div onClick={() => handleVideoSelect(video)} className="img1-wrap">
+        <Image src={video.fields.thumbnail} />
+        <div className="overlay">
+            <div className="text">{video.fields.name}</div>
+        </div>
+        </div>
+    </Slide>
+});
+
+    return (    <CarouselProvider
+            visibleSlides={4}
+            totalSlides={videos.length}
+            step={2}
+            naturalSlideWidth={400}
+            naturalSlideHeight={500}
```

Comment:
This resizing of the thumbnails doesn't keep the original video aspect ratio. At the moment, the thumbnails are created with a [320,  320 x 16/9] size in the  [backend](https://github.com/Web-Multi-Media/HttpStreamingServer/blob/master/backend/StreamServerApp/utils.py#L134). So I think we should use the same height on the frontend.

Reply:
done